terrawrap/models/pipeline.py

```diff
@@ -25,14 +26,18 @@ def __init__(self, command, config_path):
 
         with open(config_path) as config_file:
             reader = csv.DictReader(config_file)
-            entries = defaultdict(list)
+            entries = defaultdict(lambda: ([], []))
```

Comment:
A comment would be good to explain why this lambda is here

Reply:
nitpick: some sort of data structure like a named tuple instead of a tuple of lists